packages/components/src/components/drawer/drawer.jsx

```diff
@@ -54,9 +55,19 @@ class Drawer extends React.PureComponent {
                 })}
             >
                 <div className='dc-drawer__container'>
-                    {header && <div className='dc-drawer__header'>{header}</div>}
-                    <div className='dc-drawer__content'>{children}</div>
-                    {footer && <div className='dc-drawer__footer'>{footer}</div>}
+                    {header && (
+                        <div className='dc-drawer__header' style={{ height: `${headerHeight || 20}%` }}>
+                            {header}
+                        </div>
+                    )}
+                    <div className='dc-drawer__content' style={{ height: `${contentHeight || 60}%` }}>
+                        {children}
+                    </div>
+                    {footer && (
+                        <div className='dc-drawer__footer' style={{ height: `${footerHeight || 20}%` }}>
```

Comment:
Could we leave the styles up to the consuming app to decide? i.e. don't explicitly set styles here, get the app to pass them in.

Reply:
I have refactor everything.